### PR TITLE
Prerelease collector v0.51.0

### PR DESCRIPTION
**Pull request by bogdandrutu**
Fixes https://github.com/open-telemetry/opentelemetry-collector/issues/5338

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -67,6 +67,7 @@ github.com/bgentry/speakeasy v0.1.0/go.mod h1:+zsyZBPWlz7T6j88CTgSN5bM796AkVf0kB
 github.com/cenkalti/backoff/v4 v4.1.3 h1:cFAlzYUlVYDysBEH2T5hyJZMh3+5+WCBvSnK6Q8UtC4=
 github.com/cenkalti/backoff/v4 v4.1.3/go.mod h1:scbssz8iZGpm3xbr14ovlUdkxfGXNInqkPWOWmG2CLw=
 github.com/census-instrumentation/opencensus-proto v0.2.1/go.mod h1:f6KPmirojxKA12rnyqOA5BBL4O983OfeGPqjHWSTneU=
+github.com/cespare/xxhash v1.1.0 h1:a6HrQnmkObjyL+Gs60czilIUGqrzKutQD6XZog3p+ko=
```
Comment (codeboten) [inline go.sum:70]: Is this change related?

Comment (bogdandrutu) [reply go.sum]: not really, the tool did it. Will fix makefile in a separate PR.

**Review by codeboten — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,12 +4,20 @@
...
+
 - Remove deprecated model module, everything is available in `pdata` and `semconv`. (#5281)
   - Old versions of the module are still available, but no new versions will be released.
 - Remove deprecated LogRecord.Name field. (#5202)
 
-### 🚩 Deprecations 🚩
-
 ### 💡 Enhancements 💡
 
 - `pdata`: Expose `pcommon.NewSliceFromRay` and `pcommon.Slice.AsRaw` functions (#5298)
```
Comment (Aneurysm9) [inline CHANGELOG.md:23]:
```suggestion
- `pdata`: Expose `pcommon.NewSliceFromRaw` and `pcommon.Slice.AsRaw` functions (#5298)
```

Comment (Aneurysm9) [reply CHANGELOG.md:23]: Also check the PR# here, I don't believe this is correct.

**Review by Aneurysm9 — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (bogdandrutu) [reply CHANGELOG.md]: Thanks, done.
